java-client/session/src/test/java/io/deephaven/client/impl/UpdateBySpecBuilderTest.java

```diff
@@ -81,6 +81,12 @@ public UpdateByColumn.UpdateBySpec visit(CumProdSpec p) {
                     .build();
         }
 
+        // TODO: add this correctly (DHC #3666)
+        @Override
+        public UpdateByColumn.UpdateBySpec visit(DeltaSpec spec) {
+            return null;
```

Comment:
Is this the right answer? Or should we throw a UOE?

Reply:
Not sure, but It's consistent with the other operators that have merged.  `visit()` is never called for the new operators because tests can't be written until the `.proto` catches up.